benchmark/gcbench/vdparser.extra/vdc/ast/node.d

```diff
@@ -74,7 +74,7 @@ class NodeAllocData
             for(byte* p = beg; p < end && *cast(size_t*)p != 0; p += sz)
             {
                 Node n = cast(Node) p;
-                sz = n.classinfo.init.length;
+                sz = typeid(n).initializer.length;
```

Comment:
Note: I additionally changed `init` to `initializer` here

Reply:
Looks good, but this code is versioned out anyway (`.init` doesn't compile since 2.074).